pkg/restapi/rp/operation/models.go

```diff
@@ -16,8 +16,9 @@ import (
 
 // GetPresentationRequestResponse API response of getPresentationRequest.
 type GetPresentationRequestResponse struct {
-	PD  *presexch.PresentationDefinitions `json:"pd"`
-	Inv *outofband.Invitation             `json:"invitation"`
+	PD         *presexch.PresentationDefinitions `json:"pd"`
+	Inv        *outofband.Invitation             `json:"invitation"`
+	Governance json.RawMessage                   `json:"governance,omitempty"`
```

Comment:
Wondering if having a suffix like `Credential` makes it clear that its a VC,

Reply:
done